src/components/MultiTypeInput/MultiTypeInput.tsx

```diff
@@ -47,6 +47,7 @@ export interface ExpressionAndRuntimeTypeProps<T = unknown> extends Omit<LayoutP
   name: string
   disabled?: boolean
   mini?: boolean
+  showMultiTypeInputMenu?: boolean
```

Comment:
Instead of using the boolean prop lets use, if we get allowed type is Fixed, then just don't show the menu at all